pkg/cmd/install/hubaddon/scenario/resources.go

```diff
@@ -7,3 +7,56 @@ import (
 
 //go:embed addon
 var Files embed.FS
+
+var (
+	PolicyFrameworkConfigFiles = []string{
```

Comment:
I really like this new configuration file. What would you think of creating a `struct` and a map above it that also has the addon name as a key so that the install/uninstall doesn't need to specify names, something like:
```golang
var addonDeployments = map[string]AddonDeployment{
	"governance-policy-framework": {
		ConfigFiles: PolicyFrameworkConfigFiles,
		DeploymentFiles: PolicyFrameworkDeploymentFiles,
	},
	"application-manager": {
		ConfigFiles: AppManagerConfigFiles,
		DeploymentFiles: AppManagerDeploymentFiles,
	},
}

type AddonDeployment struct {
	ConfigFiles []string
	DeploymentFiles []string
}
```